core/arch/arm/plat-corstone1000/conf.mk

```diff
@@ -34,7 +34,7 @@ CFG_TEE_CORE_NB_CORE ?= 1
 CFG_TZDRAM_START ?= 0x02002000
 
 # TEE_RAM (OPTEE kernel + DATA) + TA_RAM = 3MB
```

Comment:
Please fix the comment. I would remove the "= 3MB". And while you're at it: `s/OPTEE kernel/OP-TEE kernel/`.

Reply:
Thank you.
I removed "= 3MB".
I fixed s/secure ram/secure RAM/ and s/OPTEE kernel/OP-TEE kernel/.